specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -788,6 +950,109 @@
           "description": "URL to get the next set of operation list results if there are any."
         }
       }
+    },
+    "VirtualNetworkPeering": {
+      "properties": {
```

Comment:
>properties" [](start = 7, length = 11)

please indicate what properties are required

Reply:
added required attribute for necessary properties